src/platform/errors.ts

```diff
@@ -10,10 +11,19 @@ export const translateError = (err: Error | number | string): string => {
 	if (err instanceof InternalRequestError) {
 		return 'Server Error';
 	}
-	if (err instanceof Error) {
-		return err.message;
+	let message;
+	if (err instanceof HttpError) {
+		captureException(
+			new Error(),
+			'Translating a HttpError, this should go via handleHttpErrors instead',
+		);
+		message = err.message;
+	} else if (err instanceof Error) {
+		message = err.message;
+	} else {
+		message = `${err}`;
 	}
-	return `${err}`;
+	return escapeHtml(message);
```

Comment:
I hope that this doesn't make the API generated errors look weird in the dashboard.

Reply:
I would expect not, but if so we can handle those as we spot them :)